src/rpc/rawtransaction.cpp

```diff
@@ -1367,6 +1367,141 @@ UniValue converttopsbt(const JSONRPCRequest& request)
     return EncodeBase64(ssTx.str());
 }
 
+UniValue utxoupdatepsbt(const JSONRPCRequest& request)
```

Comment:
why not do the include changes? https://github.com/bitcoin/bitcoin/pull/13932/files#diff-a58e7bb9d9a8a0287c0b7281d99da4e79b6f8c2a5780c24c6d76c14212c48640L9-L33

Also, why is this one partial?

Reply:
First one is resolved, second one is because `analyzepsbt` has not been implemented as it would require quite a lot of changes and there already exists a backlog of that